portfolionav.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
+	<title>Portfolio Page</title>
+</head>
+<body>
+	<nav>
+		 <div class="name">
+	        <ul>
+				<li>Isabelle Cochrane</li>
+			</ul>
```

Comment:
Would suggest using something other than a ul and li here. It certainly does the job without any issues but isn't reallllllly what those elements were intended for.

Using one of the text tags like h1-6 or even a span/p would be ok!